internal/connectors/plugins/public/increase/payments.go

```diff
@@ -0,0 +1,85 @@
+package increase
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/formancehq/payments/internal/models"
+)
+
+// Using pollingState from accounts.go
+
+func (p *Plugin) FetchNextPayments(ctx context.Context, req models.FetchNextPaymentsRequest) (models.FetchNextPaymentsResponse, error) {
+	state, err := p.getPollingState(req.State)
+	if err != nil {
+		return models.FetchNextPaymentsResponse{}, fmt.Errorf("failed to get polling state: %w", err)
+	}
+
+	if state.LastFetch.Add(p.config.PollingPeriod).After(time.Now().UTC()) {
+		return models.FetchNextPaymentsResponse{}, nil
+	}
```

Comment:
This is not needed. Temporal is responsible for the polling scheduling, so we should not add extra logic to skip runs.